### PR TITLE
Add new folder for MTL

### PR DESCRIPTION
**Pull request by Sathvik040105**
added MTL code for species identification (audio)

### CODE REVIEW

File: MTL/mtl_species_classi.py
Comment (Nagasai561) [inline MTL/mtl_species_classi.py:1]: Get species from a file. Don't include directly in a python.

Comment (Sathvik040105) [reply MTL/mtl_species_classi.py:1]: ok

File: MTL/mtl_species_classi.py [hunk 1]
```diff
@@ -0,0 +1,614 @@
...
+from torchaudio.transforms import Resample
+import torchaudio.transforms as T
+import warnings
+warnings.filterwarnings("ignore")
+
+def mtl_species_classi(file_path):
+    # Check if GPU is available and set the device accordingly
+    device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+
+    species = ['asbfly',
```
Comment (Nagasai561) [inline MTL/mtl_species_classi.py:27]: Put this in a file

File: MTL/mtl_species_classi.py [hunk 2]
```diff
@@ -0,0 +1,614 @@
...
+    'whiter2',
+    'whrmun',
+    'whtkin2',
+    'woosan',
+    'wynlau1',
+    'yebbab1',
+    'yebbul3',
+    'zitcis1'] ; num_species_classes = len(species)
+
+    common_names = {'wbbfly1': 'White-bellied Blue Flycatcher',
```
Comment (Nagasai561) [inline MTL/mtl_species_classi.py:210]: Put this also in a file.

File: MTL/mtl_species_classi.py [hunk 3]
```diff
@@ -0,0 +1,614 @@
...
+    'Passeriformes',
+    'Pelecaniformes',
+    'Piciformes',
+    'Columbiformes',
+    'Psittaciformes',
+    'Ciconiiformes',
+    'Suliformes',
+    'Cuculiformes'] ; num_order_classes = len(orders)
+
+    family = ['Scolopacidae (Sandpipers and Allies)',
```
Comment (Nagasai561) [inline MTL/mtl_species_classi.py:413]: This also

**Review by Nagasai561 — CHANGES_REQUESTED** [review]
Everything seems good. Just add those names in a text file.

**Pull request closed without merging**